contracts/cw-ics721-bridge/Cargo.toml

```diff
@@ -15,14 +15,14 @@ library = []
 [dependencies]
 cosmwasm-std = { version = "1.1", features = ["ibc3"] }
 cosmwasm-schema = "1.1"
-cw-storage-plus = "1.0.1"
+cw-storage-plus = { rev = "6db957ce730a95141a3ab4dc5ab76fb38e8c0c42", git = "https://github.com/CosmWasm/cw-storage-plus" }
```

Comment:
can you clarify why we need this hardcoded version of cw storage plus?

Reply:
I assume its to include this: https://github.com/CosmWasm/cw-storage-plus/pull/30 
@taitruong